components/lifter.py

```diff
@@ -66,24 +102,29 @@ def move_to_encoder_pos(self, input_setpos):
             input_setpos (int): Encoder position to move lift to in encoder counts
         """
         self.set_pos = input_setpos
-        self.lift_motor.set(mode=self.lift_motor.ControlMode.MotionMagic, value=self.set_pos)
 
     def get_pos(self):
         """Returns encoder position on lift
 
        Returns:
             int: The location of the lift
         """
-        return self.lift_motor.getSelectedSensorPosition(0)
+        return self.motor.getSelectedSensorPosition(0)
 
     def at_pos(self):
         """Finds if cube location is at setops and within threshold
 
-        Args:
-            pos (int): The target encoder position
-
         Returns:
             bool: If the encoder is at the pos
         """
         current_pos = self.get_pos()
         return current_pos - self.THRESHOLD <= self.set_pos <= current_pos + self.THRESHOLD
```

Comment:
`abs(self.set_pos - current_pos) < self.THRESHOLD` - much more readable

Reply:
Heh. This was originally written as `self.set_pos >= current_pos - self.THRESHOLD and self.set_pos <= current_pos + self.THRESHOLD`, so it's an improvement at least.